web/src/pages/CreateRoom/hooks/useCreateRoom.ts

```diff
@@ -0,0 +1,82 @@
+import { createStore } from "solid-js/store";
+
+import post from "../../../services/post";
+
+type CreateRoomFields = {
+	moderatorName: string;
+	selectedOptions: Number[];
+	voterOptions: Number[];
+	numberOfOptions: Number;
+	error: string | null;
+};
+
+const defaultStoreData = {
+	moderatorName: "",
+	selectedOptions: [],
+	voterOptions: [],
+	numberOfOptions: 5,
+	error: null,
+};
+
+const options = {
+	fibonacci: [1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987],
```

Comment:
After hardcoding this in here it made me think we should stores these on the backend and just pass along identifiers for which option to pick. Not sure if ya'll discussed this or not thought? @misterbastean @sarashunter ?

Reply:
If we keep it on the BE we can reduce the Create Room message body size slightly, but it's a trivial amount (and an already tiny body). 

On the other hand, doesn't this mean we'd need to query the BE to get the values when this page loads each time, in order to display the "Final Preview" section? If that's the case, I would think hard-coding on the FE is best in order to eliminate that extra call.

Reply:
Frontend makes sense to me, but it doesn't matter too much.

I pity the person who has to work a story with 987 points D:

Reply:
Cool sounds good to me, I'll leave this as is

Reply:
Instead of hardcoding these values, could we calculate them instead? If we know the sequence (ie the formula) and the number of iterations, could just calculate the series and use that instead?

Reply:
It would theoretically make it easier to add more sequences or change the range of the number of options slider in the future. If we think the ROI on that is too low for now, then I'm fine with kicking it down the road a bit. Could add a bit of flare for the pitch though 👀 .